tools/ci_build/github/azure-pipelines/templates/use-android-ndk.yml

```diff
@@ -0,0 +1,30 @@
+# Installs the Android NDK and sets environment variables ANDROID_NDK_HOME and ANDROID_NDK_ROOT to refer to it.
+
+  parameters:
+  - name: AndroidNdkVersion
+    type: string
+    default: "25.0.8775105"  # LTS version
+
+  steps:
+  - bash: |
+      set -e
+
+      "${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager" --install "ndk;${{ parameters.AndroidNdkVersion }}"
```

Comment:
What if the sdk is already installed? Would it cause any error?

Reply:
tested locally, was able to run it multiple times without error

Reply:
Is it possible to check if the version is already installed? This way when version 25 is added to the hosted agents we dont need to make any changes.

Reply:
I think the installation is a hot fix. It should be removed while the upstream image become stable.

Reply:
I think the installation is a hot fix. It should be removed while the upstream image become stable.

Reply:
we may not need to make changes when 25 is already installed on the hosted agent if running the install command again succeeds.
we can update it to not attempt to install anything at all eventually if that is desired, but I would like to have the NDK version that we actually use be more visible.